outline.md

```diff
@@ -0,0 +1,17 @@
+## Motivation
+
+Introduce the document by describing the motivations for producing a consolidated set of software citation principles. This section is basically an overview of the different use cases/scenarios where it would be desirable to have a way to cite software.
+
```

Comment:
I like the outline a lot. In addition to motivation, I think perhaps we could have some survey of factors (psychological, cultural, or technological, etc.) hindering software citation.